cli/lib/kontena/cli/helpers/health_helper.rb

```diff
@@ -49,5 +49,49 @@ def node_health(node, grid_health)
         return node['connected'] ? :ok : :offline
       end
     end
+
+    # @return [Symbol]
+    # @return [String]
+    def node_etcd_health(node_health)
```

Comment:
Moved the `time_since` to a separate `Kontena::Cli::Helpers::TimeHelper`.

The `node_etcd_health` is still there in the `HealthHelper`, because I think that's the right place for it.